system/flatcar-linux-update-agent/templates/update-agent.yaml

```diff
@@ -1,35 +1,38 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: container-linux-update-agent
+  name: flatcar-linux-update-agent
   namespace: kube-system
 spec:
   updateStrategy:
     type: RollingUpdate
+  selector:
+    matchLabels:
+      app: flatcar-linux-update-agent
   template:
     metadata:
       labels:
-        app: container-linux-update-agent
-        container-linux-update.v1.coreos.com/agent-version: v0.7.0
+        app: flatcar-linux-update-agent
+        agent-version: v0.7.3
```

Comment:
Not sure this label (and annotation) adds much other than to drift from the image that is updated eventually while this value is forgotten.